### DISCUSSION TITLE
ToonSquid cameo on tech review video

### DISCUSSION THREAD

**Discussion by budaloops** [keiwando/toonsquid#631] [General]
Since the announcement of the 2.0 closed beta, 

I've been excited about the new features. I've been searching for 'ToonSquid' content on YouTube, but it's still early unless we start creating some. In the meantime, I've been exploring the App Store for new creative apps and tools. I just stumbled upon a video of a tech YouTuber casually using ToonSquid on his iPad, which was a pleasant surprise since I haven't seen it used outside of illustration or animation videos. 

https://github.com/user-attachments/assets/64a77b88-863f-484b-a6cb-f47ff16d9d5d

I thought I'd share it here, as I'm looking forward to seeing new posts about 2.0, so I might as well make a new post myself. Anyway, I'm excited about ToonSquid, but now I need to get back to work—I've procrastinated long enough!

**Comment by DWrightAnimation** [comment]
I regret to inform you, that wasn't ToonSquid, he was using Adobe Animate.

**Reply by budaloops** [reply]
Damn. There’s too many tools on the UI. I got all excited. Thanks D 😄
